### PR TITLE
Minor fixes for the Compatibility Matrix Test

### PR DESCRIPTION
**Pull request by toninis**
#### Summary

- Introduced proper matrix for CMT testing

#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] Has UI changes
- [ ] read and understood our [Contributing Guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
- [ ] completed [Mattermost Contributor Agreement](https://mattermost.com/contribute/)
- [ ] executed `npm run lint:js` for proper code formatting
- [ ] Run E2E tests by adding label `Run Desktop E2E Tests`

#### Device Information
This PR was tested on: 

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: .github/workflows/compatibility-matrix-testing.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ name: Compatibility Matrix Testing
 on:
   workflow_dispatch:
     inputs:
-      MM_TEST_SERVER_URL:
-        description: "The test server URL"
+      CMT_MATRIX:
+        description: "A JSON object representing the testing matrix"
```
Comment (spirosoik) [inline .github/workflows/compatibility-matrix-testing.yml:7]: is this needed?

**Review by spirosoik — COMMENTED** [review]
one minor comment.

**Review by yasserfaraazkhan — APPROVED** [review]

File: .github/workflows/compatibility-matrix-testing.yml (hunk 1 above)
Comment (saturninoabril) [reply .github/workflows/compatibility-matrix-testing.yml:7]: Probably here - https://github.com/mattermost/desktop/pull/2994/files#diff-8271618e3df5dba8c8c5470a34f16e8ccbefa86b4ddf9b46a689709dfca32dc0R78

**Review by saturninoabril — APPROVED** [review]
Thanks @toninis!

**Pull request merged**